node/cn/handler.go

```diff
@@ -1117,8 +1117,8 @@ func (pm *ProtocolManager) BroadcastTxs(txs types.Transactions) {
 	// This function calls sendTransaction() to broadcast the transactions for each peer.
 	// In that case, transactions are sorted for each peer in sendTransaction().
 	// Therefore, it prevents sorting transactions by each peer.
-	if !sort.IsSorted(types.TxByPriceAndTime(txs)) {
-		sort.Sort(types.TxByPriceAndTime(txs))
+	if !sort.IsSorted(types.TxByTime(txs)) {
```

Comment:
1. `gasPrice` check first, and early arrived tx first if the `gasPrice` is the same in the prev implementation.
2. Early arrived tx first in this change.

This change seems not related to the new policy of the tx pool. Lack of motivation for this change in the description of the PR.